setup-env/src/actionInput/index.js

```diff
@@ -30,6 +33,11 @@ class ActionInput {
       // non-compulsory fields
       this.buildName = core.getInput(INPUT.BUILD_NAME);
       this.projectName = core.getInput(INPUT.PROJECT_NAME);
+      this.githubToken = core.getInput(INPUT.GITHUB_TOKEN);
+      this.githubApp = core.getInput(INPUT.GITHUB_APP);
+      this.rerunAttempt = process?.env?.GITHUB_RUN_ATTEMPT;
```

Comment:
check if available in github context

Reply:
Done, github context does not have rerunAttempt, runId but process does, using that for consistency now. It also removes the import for github

Reply:
https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs#example-printing-context-information-to-the-log:~:text=the%20workflow%20run.-,github.run_attempt,-string

lets check above once.

Reply:
Using process.env as context is available at action level which doesn't have these details. Accessing from Process.env